db/seeds.d/17-notification_blueprints.rb

```diff
@@ -0,0 +1,30 @@
+# seeds UI notification blueprints that are supported by Foreman.
+class NotificationSeeds
+  attr_reader :attributes
+
+  # needed in order to resolve rails urls for notifications
+  include Rails.application.routes.url_helpers
+  def initialize(blueprint_attributes)
+    @attributes = blueprint_attributes
+  end
+
+  def configure
+    blueprint = NotificationBlueprint.find_by(name: attributes[:name])
+    if blueprint
+      blueprint.update_attributes!(attributes)
+    else
+      NotificationBlueprint.create!(attributes)
+    end
+  end
+end
+
+blueprints = [
+  {
+    group: _('Provisioning'),
+    name: 'host_build_completed',
+    message: _('%{subject} has been provisioned successfully'),
```

Comment:
This is missing `{ :actions => { :links => hosts_path(subject) } ` (probably you'll have to wrap `hosts_path` in a lambda or something though)